File parameter only accepts relative file paths

While I was coding on the CLI code, I tried to use absolute config file path to test my local CLI code against another project that uses pgtyped.

However, the absolute file path that I provided was still concatenated with current working path:

```
$ DEBUG=* node lib/index.js -c /Users/golergka/Projects/some/other/project/pgtyped.json
Failed to parse config file:
Cannot find module '/Users/golergka/Projects/pgtyped/packages/cli/Users/golergka/Projects/some/other/project/pgtyped.json'
```

(Another project path was changed to "some/other/project").

--- a/packages/cli/src/config.ts
+++ b/packages/cli/src/config.ts
@@ -1,7 +1,7 @@
 /** @fileoverview Config file parser */
 
 import * as Either from 'fp-ts/lib/Either';
-import { join } from 'path';
+import { join, isAbsolute } from 'path';
 import * as t from 'io-ts';
 import { reporter } from 'io-ts-reporters';
 
@@ -71,7 +71,10 @@ function merge<T>(base: T, ...overrides: Partial<T>[]): T {
 }
 
 export function parseConfig(path: string): ParsedConfig {
-  const configObject = require(join(process.cwd(), path));
+  if (!isAbsolute(path)) {
+    path = join(process.cwd(), path);
+  }
+  const configObject = require(path);
   const result = configParser.decode(configObject);
   if (Either.isLeft(result)) {
     const message = reporter(result);

--- a/packages/cli/src/config.ts
+++ b/packages/cli/src/config.ts
@@ -71,10 +71,8 @@ function merge<T>(base: T, ...overrides: Partial<T>[]): T {
 }
 
 export function parseConfig(path: string): ParsedConfig {
-  if (!isAbsolute(path)) {
-    path = join(process.cwd(), path);
-  }
-  const configObject = require(path);
+  const fullPath = isAbsolute(path) ? path : join(process.cwd(), path);
+  const configObject = require(fullPath);
   const result = configParser.decode(configObject);
   if (Either.isLeft(result)) {
     const message = reporter(result);

